ingress/controllers/gce/controller/utils.go

```diff
@@ -315,3 +320,40 @@ func (t *GCETranslator) toNodePorts(ings *extensions.IngressList) []int64 {
 	}
 	return knownPorts
 }
+
+func getZone(n api.Node) string {
```

Comment:
it will use the default zone if the key isn't found for some reason, i.e older clusters with newer image